addons/mail/static/src/core/common/store_service.js

```diff
@@ -429,12 +429,16 @@ export class Store extends BaseStore {
         const validMentions = {};
         const partners = [];
         const threads = [];
-        for (const partner of mentionedPartners) {
-            const index = body.indexOf(`@${partner.name}`);
-            if (index === -1) {
-                continue;
+        if (body.includes("@everyone")) {
```

Comment:
What if we actually wanted to mention `@everyone` user name?

Reply:
As is implemented it would mention everyone, we can add a flag indicating the presence or not of "special" mentions.